### PR TITLE
Add DevUI Card

### LINKED ISSUE #71: DevUI: Add Card
You can assign this to me. 

- [x] DevUI card displaying PostGres DB info
- [x] DevServices information about connected DB

image

### PR DESCRIPTION
**Pull request by melloware**
Fix #71 Add DevUI Card

image

### CODE REVIEW

**Comment by melloware** [comment]
@fjtirado  Do you mind reviewing and merging this before I move to the other ticket to fix Quarkus 3.6.0?

File: deployment/pom.xml [hunk 1]
```diff
@@ -27,6 +31,14 @@
 			<groupId>io.quarkus</groupId>
 			<artifactId>quarkus-datasource-deployment</artifactId>
 		</dependency>
+		<dependency>
```
Comment (fjtirado) [inline deployment/pom.xml:34]: @melloware Im probably lacking context, but adding these two dependencies here does not seem a good idea at all. If there is anyway to add the DevUI Card (why this is needed for?) without adding two unrelated dependencies that will be carry forward in any server installation using this extension.

Comment (melloware) [reply deployment/pom.xml:34]: the VertX SPI is required for Dev UI it is not an optional dependency and I am pretty sure if gets removed if you are not in `quarkus:dev` mode.  As for JAXP that was only necessary to support "runtime" getting of the running Port but if we switch to Build Time config I can remove this dependency.

Comment (fjtirado) [reply deployment/pom.xml:34]: @melloware Lets do first the switch to build time config and then retake this one

**Comment by melloware** [comment]
@fjtirado this PR is also ready.  IT has to stay a JSONRPC Service for serving the port to the DevUI card because the Port is dynamically allocated at runtime not at build time.

But this will allow me to do a future PR where we also provide the PGAdmin UI from the Dev Card

File: runtime/pom.xml [hunk 2]
```diff
@@ -50,6 +55,10 @@
       <groupId>io.quarkus</groupId>
       <artifactId>quarkus-datasource</artifactId>
     </dependency>
+    <dependency>
+      <groupId>io.quarkus</groupId>
+      <artifactId>quarkus-jaxp</artifactId>
```
Comment (fjtirado) [inline runtime/pom.xml:60]: Is there any way to avoid adding jaxp dependency at runtime?

Comment (fjtirado) [reply runtime/pom.xml:60]: My main concern is that not all users adding the dependency (embedded sql in memory) might be interested on carrying forward a UI related dependency.

Comment (fjtirado) [reply runtime/pom.xml:60]: Do you know if jaxp carrry forward more dependencies or it is just the jaxp?

**Comment by melloware** [comment]
Any chance we can get a release put out so I can move off Quarkus 3.2.9 in my project?

**Pull request merged**